roles/foreman_devel/tasks/check_forks_of_repos.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: "Check forks of user's repository"
+  uri:
+    url: "https://github.com/{{ katello_devel_github_username }}/{{ item.split('/') | last }}"
+    no_log: True
+  with_items: "{{ foreman_installer_options_internal_use_only }}"
```

Comment:
This looks like a coupling between roles and I'm not sure that's a good thing. Also note that we have the normal installer options users can specify but that's not an array so harder to use in this way.

Reply:
Yes, I understand. This is based on https://github.com/theforeman/forklift/pull/623.
This is really important for newone (as am I), who does not have all forks of gits and the install proces fails in the middle. The better way is checking all requirements in the beginning.